server/py/framework/db/sqldb/db.py

```diff
@@ -6811,40 +6822,22 @@ def store_model_endpoint(
         self,
         session,
         model_endpoint: mlrun.common.schemas.ModelEndpoint,
-        name: str,
-        function_name: str,
-        project: str,
     ) -> mlrun.common.schemas.ModelEndpoint:
+        if not model_endpoint.metadata.name or not model_endpoint.metadata.project:
+            raise mlrun.errors.MLRunInvalidArgumentError(
+                "Model endpoint name and project must be provided"
```

Comment:
when failing on multile conditions it's better to print the relevant values - otherwise the user does not which of the reasons caused the exception.